packages/ckeditor5-engine/src/model/documentselection.js

```diff
@@ -619,6 +632,11 @@ class LiveSelection extends Selection {
 		// @type {Set}
 		this._overriddenGravityRegister = new Set();
 
+		// Prefixes of marker names that should affect `LiveSelection#markers` collection.
+		// @private
+		// @type {Array.<String>}
+		this._observedMarkerGroups = [];
```

Comment:
How about using `Set` for potential dedups?

Reply:
done